widget/richtext_objects.go

```diff
@@ -364,7 +364,7 @@ type RichTextStyle struct {
 	Alignment fyne.TextAlign
 	ColorName fyne.ThemeColorName
 	Inline    bool
-	SizeName  fyne.ThemeSizeName
+	SizeName  fyne.ThemeSizeName // The theme name of the text size to use, if blank will be the standard text size
```

Comment:
I actually don't know - will GoDoc and IDEs pick this up if the comment is inline or does it have to be above?

Reply:
Yes both are picked up (I just did a quick check).
Many of our widgets just use this inline format (often the way until a Since line is needed).